libs/vm_modules/tests/unit/ml.cpp

```diff
@@ -492,4 +492,43 @@ TEST_F(MLTests, optimiser_set_graph_test)
   ASSERT_TRUE(toolkit.Run());
 }
 
+TEST_F(MLTests, graph_step_test)
+{
+  static char const *src = R"(
+    function main() : Fixed64
```

Comment:
you can return the loss difference tensor instead of fixedpoint64 value.
This is preferable because Fixed64 is a (necessary) hard-coding of the value set in vm_modules/math/base_types.hpp
If we decide to replace this base type ever (quite likely we would replace it with fp128 when available), we will need to update every hard-coded value